Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -1084,7 +1088,20 @@ - (void)updateHtmlLayoutWithContext:(LPActionContext *)context
         _popupGroup.frame = CGRectMake(htmlX, htmlY, htmlWidth, htmlHeight);
         
     } else if (isIPhoneX) {
-        _popupGroup.frame = CGRectMake(0, -statusBarHeight, screenWidth, screenHeight+statusBarHeight+bottomSafeAreaHeight);
+        // Do not offset the bottom safe area (control panel) on landscape.
+        // Counter the status bar's height twice to center it.
```

Comment:
Could you help me understand why counting left twice helps center this?

Reply:
this has been quite a mystery for me too. It took me sometime to get to the bottom of it.
On landscape the 44 insets is set on both left and right.

For whatever reason if i take the right (or left) offset from both X and Width, the layout gets screwed up. Feel free to checkout my branch and play around with it. 

For more info:
popGroup.frame = {{-88, 0}, {900, 375}}
iPhoneX dimension = 812, 375